test/compflags/checks/checks.chpl

```diff
@@ -0,0 +1 @@
+writeln(boundsChecking, castChecking, chpl_checkNilDereferences, chpl_checkDivByZero);
```

Comment:
As a general style note, we often handle cases like this using:

```chapel
writeln((arg1,arg2,arg3));
```

so that the output ends up being `(true, true, true, true)` rather than getting glommed together into one string.  Clearly, either will accomplish what the test is intending (i.e., I wouldn't hold up the PR over this), but it makes for a slightly nicer output result.

Reply:
Yes it's nicer!